File: app/src/main/java/org/test/kotlin/demo/api/dto/UserDTO.java
package org.test.kotlin.demo.api.dto;

import com.google.gson.annotations.SerializedName;

public class UserDTO {

    @SerializedName("id")
    private long id;

    @SerializedName("login")
    private String user;

    @SerializedName("name")
    private String name;

    @SerializedName("avatar_url")
    private String avatarURL;

    @SerializedName("html_url")
    private String profileURL;

    @SerializedName("public_repos")
    private int repositories;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAvatarURL() {
        return avatarURL;
    }

    public void setAvatarURL(String avatarURL) {
        this.avatarURL = avatarURL;
    }

    public String getProfileURL() {
        return profileURL;
    }

    public void setProfileURL(String profileURL) {
        this.profileURL = profileURL;
    }

    public int getRepositories() {
        return repositories;
    }

    public void setRepositories(int repositories) {
        this.repositories = repositories;
    }

}

File: app/src/main/java/org/test/kotlin/demo/api/dto/UserDTO.kt
package org.test.kotlin.demo.api.dto

import com.google.gson.annotations.SerializedName

data class UserDTO(
        @SerializedName("id") val id: Long,
        @SerializedName("login") val user: String,
        @SerializedName("name") val name: String,
        @SerializedName("avatar_url") val avatarURL: String,
        @SerializedName("html_url") val profileURL: String,
        @SerializedName("public_repos") val repositories: Int
)
